src/phoenix/experimental/callbacks/langchain_tracer.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
+from uuid import uuid4
+
+from langchain.callbacks.tracers.base import BaseTracer
+from langchain.callbacks.tracers.schemas import Run
+
+from phoenix.trace.schemas import Span, SpanContext, SpanKind, SpanStatusCode
+
+
+class OpenInferenceTracer(BaseTracer):  # type: ignore
+    def __init__(self) -> None:
+        self.spans: List[Span] = []
+        self.trace_id = uuid4()
```

Comment:
a trace_id needs to be unique per "run" - this seems strange to have this attached to the tracer?

Reply:
revised to use existing Tracer class